packages/amplify-util-uibuilder/commands/cloneComponentsFromEnv.js

```diff
@@ -15,63 +15,60 @@ async function run(context) {
-      context,
-      newEnvName,
-      appId,
-    ]);
+  const appId = args.appId ? args.appId : context.exeInfo.teamProviderInfo[environmentName].awscloudformation.AmplifyAppId;
 
-    if (process.env.UI_BUILDER_ENDPOINT) {
-      awsConfigInfo.endpoint = process.env.UI_BUILDER_ENDPOINT;
-    }
+  const awsConfigInfo = await context.amplify.invokePluginMethod(context, 'awscloudformation', undefined, 'loadConfigurationForEnv', [
+    context,
+    newEnvName,
+    appId,
+  ]);
 
-    if (process.env.UI_BUILDER_REGION) {
-      awsConfigInfo.region = process.env.UI_BUILDER_REGION;
-    }
+  if (process.env.UI_BUILDER_ENDPOINT) {
+    awsConfigInfo.endpoint = process.env.UI_BUILDER_ENDPOINT;
+  }
 
-    const amplifyUIBuilder = new aws.AmplifyUIBuilder(awsConfigInfo);
+  if (process.env.UI_BUILDER_REGION) {
+    awsConfigInfo.region = process.env.UI_BUILDER_REGION;
+  }
 
-    const components = existingComponents.entities;
-    if (!components.length) {
-      logger.info(`No UIBuilder components found in app ${appId} from env ${sourceEnvName}. Skipping component clone process.`);
-      return;
-    }
-    const promises = components.map(async component => {
-      return await amplifyUIBuilder
-        .createComponent({
-          appId,
-          environmentName,
-          componentToCreate: {
-            bindingProperties: component.bindingProperties,
-            children: component.children,
-            componentType: component.componentType,
-            name: component.name,
-            overrides: component.overrides,
-            properties: component.properties,
-            sourceId: component.sourceId,
-            variants: component.variants,
-          },
-        })
-        .promise();
-    });
-    await Promise.all(promises);
+  const amplifyUIBuilder = new aws.AmplifyUIBuilder(awsConfigInfo);
 
-    console.log(
-      `Successfully cloned ${components.length} UIBuilder components in app ${appId} from env ${sourceEnvName} to env ${newEnvName}.`,
-    );
-  } catch (e) {
-    logger.warn('This endpoint is broken for now:');
-    logger.error(e);
+  const components = existingComponents.entities;
+  if (!components.length) {
+    printer.debug(`No UIBuilder components found in app ${appId} from env ${sourceEnvName}. Skipping component clone process.`);
+    return;
   }
+  const promises = components.map(async component => {
+    return await amplifyUIBuilder
+      .createComponent({
+        appId,
+        environmentName,
+        componentToCreate: {
+          bindingProperties: component.bindingProperties,
+          children: component.children,
+          componentType: component.componentType,
+          name: component.name,
+          overrides: component.overrides,
+          properties: component.properties,
+          sourceId: component.sourceId,
+          variants: component.variants,
+        },
+      })
+      .promise();
+  });
+  await Promise.all(promises);
+
+  console.log(
```

Comment:
printer.debug?

Reply:
I think this one is `printer.info`. We actually _do_ want to show customers how many components they generated